trails-viz-api/trailsvizapi/config/app_config.py

```diff
@@ -20,6 +20,7 @@
     # 'Mount Baker-Snoqualmie General Forest': 'MBS_GFA'
     'East Cascades': 'EastCascades',
     'Coronado': 'Coronado'
```

Comment:
The 'Coronado' line also needs a comma after it if you are adding a new line after (each line should end with a comma).